src/Tests/MessageUiMassiveHardCodedArguments.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drupal\message_ui\Tests\MessageUiMassiveHardCodedArguments.
+ */
+
+namespace Drupal\message_ui\Tests;
+
+use Drupal\message\Tests\MessageTestBase;
+use Drupal\message\Entity\Message;
+
+/**
+ * Testing the update of the hard coded arguments in massive way.
+ *
+ * @group Message UI
+ */
+class MessageUiMassiveHardCodedArguments extends MessageTestBase {
+
+  /**
+   * The user object.
+   * @var
```

Comment:
Add a line and specify the type of the property. it's probably `AccountInterface` or something similar.